frontend/public/components/modals/_modals.scss

```diff
@@ -93,13 +93,20 @@
 .modal-dialog {
   margin: 10px;
   margin-bottom: 0;
+  max-width: calc(100% - var(--pf-global--spacer--xl));
   outline: 0;
   position: relative;
-  width: auto;
-
+  width: 100%;
   @media (min-width: $pf-global--breakpoint--md) {
     margin: 30px auto;
-    width: 600px;
+    &:not(.modal-lg) {
+      max-width: 600px;
+    }
+  }
+  @media (min-width: $pf-global--breakpoint--xl) {
+    &.modal-lg {
+      max-width: 70rem; // match PF --pf-c-modal-box--m-lg--lg--MaxWidth
```

Comment:
Is it not possible to use the PF var instead of a hard-coded value?

Reply:
Added comment explaining hard coded unit